tpot/tpot.py

```diff
@@ -1007,6 +1081,51 @@ def _select_fwe(self, input_df, alpha):
 
         mask_cols = list(training_features.iloc[:, mask].columns) + self.non_feature_columns
         return input_df[mask_cols].copy()
+    
+    def _ekf_mask(self, input_df, expert_source):
+        """Uses expert knowledge source to subset/mask columns of the input data
+        
+        Parameters
+        ----------
+        input_df: pandas.DataFrame {n_samples, n_features+['class', 'group', 'guess']}
+            Input DataFrame to perform feature selecton on
+        expert_source: Boolean list/mask for features provided by user
+        
+        Returns
+        -------
+        subsetted_df: andas.DataFrame {n_samples, n_filtered_features + ['guess', 'group', 'class']}
+            Returns a DataFrame containing the 'best' features provided by expert_source
+        
+        """
+#        training_features = input_df.loc[input_df['group'] == 'training'].drop(self.non_feature_columns, axis=1)
```

Comment:
`expert_source ` should be an `int` that indexes into the expert knowledge sources stored in `self.expert_source`.